### DISCUSSION TITLE
Thumbs down for thumbs down

### DISCUSSION THREAD

**Discussion by 1der1** [community/community#11388] [Discussions]
Many social media platforms removed the thumbs-down or dislike "feature" for a reason. If someone has an objection to a post, he should reply with facts that are constructive.

**Comment by svick** [comment]
I disagree. Most software-related decisions are not about hard facts, they are subjective and involve trade-offs. And giving people a simple way to express their opinion (positive or negative) makes it easier to gauge the overall opinion of the community, and prevents a flood of similar posts.
